src/index.ts

```diff
@@ -2,15 +2,30 @@
 
 import yargs from "yargs"
 import { TeamsMessaging } from "./commands/TeamsMessaging"
+import { TeamsMessagingPublish } from "./commands/TeamsMessagingPublish"
 
 async function run() {
   await yargs(process.argv.slice(2))
-    .command("teams", "This command is used to send a teams message via a passed webhook", async (args) => {
-      const teamsMessagingCommand = new TeamsMessaging()
-      const options = await teamsMessagingCommand.parseCLIOptions(args)
-      await teamsMessagingCommand.run(options)
-      return options
-    })
+    .command(
+      "teams-build",
```

Comment:
i don't like the name.. what are you building?

WTB `teams-develop` for this one and `teams-production` for the other?